app/cameratakelib/src/main/java/com/genar/hmssandbox/huawei/cameratakelib/utils/LogUtil.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2021. Explore in HMS. All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.genar.hmssandbox.huawei.cameratakelib.utils;
+
+import android.text.TextUtils;
+import android.util.Log;
+
+public class LogUtil {
+
+    public static final String TAG_PREFIX = "";
```

Comment:
all log writing have to close in production mod. should check with BuildConfig.DEBUG before logging